extractor/nft.go

```diff
@@ -0,0 +1,95 @@
+package extractor
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/likecoin/likecoin-chain-tx-indexer/db"
+	"github.com/likecoin/likecoin-chain-tx-indexer/logger"
+	"github.com/likecoin/likecoin-chain-tx-indexer/utils"
+)
+
+type nftClassMessage struct {
+	Input   db.NftClass `json:"input"`
+	Creator string      `json:"creator"`
+}
+
+func createNftClass(payload db.EventPayload) error {
+	var message nftClassMessage
+	if err := json.Unmarshal(payload.Message, &message); err != nil {
+		return fmt.Errorf("Failed to unmarshal NFT class message: %w", err)
+	}
+	var c db.NftClass = message.Input
+	c.Id = utils.GetEventsValue(payload.Events, "likechain.likenft.EventNewClass", "class_id")
```

Comment:
Please update these values containing `likenft` to `likenft.v1` in v3

Reply:
updated